libensemble/tests/unit_tests_nompi/test_aaa_comms.py

```diff
@@ -52,42 +53,42 @@ def test_qcomm_proc_terminate2():
         assert not mgr_comm.running
 
 
-def ignore_handler(signum, frame):
-    print("Ignoring SIGTERM")
-
-
-def test_qcomm_proc_terminate3():
-    "Test that a QCommProcess ignoring SIGTERM manages."
-
-    with comms.QCommProcess(worker_main_waiting, 2) as mgr_comm:
-        time.sleep(0.5)
-
-        flag = True
-        try:
-            mgr_comm.recv(timeout=0.5)
-            flag = False
-        except comms.Timeout:
-            pass
-        assert flag, "Should time out on recv"
-
-        flag = True
-        try:
-            mgr_comm.result(timeout=0.5)
-            flag = False
-        except comms.Timeout:
-            pass
-        assert flag, "Should time out on result"
-
-        flag = True
-        try:
-            mgr_comm.terminate(timeout=0.5)
-            flag = False
-        except comms.Timeout:
-            pass
-        assert flag, "Should time out on terminate"
-
-        assert mgr_comm.running, "Should still be running"
-        mgr_comm.send("Done")
+# def ignore_handler(signum, frame):
```

Comment:
Intentionally commented out?

Reply:
This is the test we removed that is problematic on CI (macOS in virtual env). We could remove it.

Reply:
We could remove, or at least add a note to the source code about why it's commented out